### ISSUE TITLE
Uploaded trained model

### ISSUE THREAD

**Issue by phongnhhn92** [jenspetersen/gqn-pytorch#1]
Thanks for your code ! Can you upload your trained model also. I would like to help to create the test function.

**Comment by jenspetersen** [comment]
Of course! but will only get to it mid next week

**Comment by jenspetersen** [comment]
My saved model appears to be broken :( I get an error that 8 bytes are missing when I try to load it. Honestly no idea how that happened... I uploaded the checkpoint to the data bucket if you want to try for yourself, if you do, let me know if you get the same error. The deserialized data should be a dict {"model": model state dict, ...} with some other entries as well.

**Comment by phongnhhn92** [comment]
Thanks ! But unfortunately, I have the same 8 bytes missing issue. May be the problem is on your computer side when you try to save the model. Anyways, I am training again with the numpy version of Shepard-Metzler-5 you have provide and the training is going well. However, I really would like to test it on another dataset from GQN datasets as well. Is this possible if you can share the code you have used to convert those tfrecords into numpy so that I can further help to to fasten the process ?

**Comment by jenspetersen** [comment]
They will be uploaded soon :) For the conversion I'm basically using this https://github.com/l3robot/gqn_datasets_translator but modified to output numpy files instead of pytorch tensors. Then you just have to merge al the single numpy files. I'll upload the conversion file with the rest of the data anyway.

**Comment by phongnhhn92** [comment]
Hello! Long time I didn't update the status and I am happy to see that you uploaded all dataset in numpy format but somehow I can not access them anymore. Is there something wrong?

**Comment by jenspetersen** [comment]
Yeah, my Google Cloud credit ran out. I already contacted the DeepMind guys if they want to host the numpy versions alongside the tfrecords, fingers crossed!

**Comment by phongnhhn92** [comment]
Thank @jenspetersen I hope people from DeepMind will reply also. Please update me with future updates!
Actually, I am building my own dataset so is this Ok if you can upload the code that you have used to convert tfrecords to numpy files? I would really appreciate the help.

**Comment by jenspetersen** [comment]
It's already there :) data/convert_to_numpy.py

**Comment by phongnhhn92** [comment]
Oh, I see! Thanks a lot, I haven't clone the latest repo so I didn't see it ahah :v

**Comment by male-lion** [comment]
@phongnhhn92
I feel very sorry for asking you a question so abruptly.
I am a college student from China, and I want to ask you a question.
I have never found a way to train. Can we communicate by email?
Do you have time?
I can reply to you at any time.
I look forward to waiting for your reply and an academic consultation with you!
I'm sorry to disturb you.
My email address is: 282117590@qq.com
What is your email address?
